types/has/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for has 1.0
+// Project: https://github.com/tarruda/has
+// Definitions by: Jordan Harband <https://github.com/ljharb>
+//                 ExE Boss <https://github.com/ExE-Boss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Determines whether an object has an own property with the specified name.
+ *
+ * @param target The object that contains the property.
+ * @param property A property name.
+ */
+declare function hasOwnProperty(target: object, property: PropertyKey): boolean;
```

Comment:
actually now that i think about it, can’t this take {}, ie, anything that’s not nullish?